spec/system/candidate_interface/candidate_new_user_with_course_params_spec.rb

```diff
@@ -125,11 +126,19 @@ def and_i_see_the_form_to_pick_a_location
   end
 
   def and_my_course_from_find_id_should_be_set_to_nil
-    candidate = Candidate.find_by!(email_address: @email)
-    expect(candidate.course_from_find_id).to eq(nil)
+    @candidate = Candidate.find_by!(email_address: @email)
+    expect(@candidate.course_from_find_id).to eq(nil)
   end
 
   def then_i_should_see_the_course_choices_site_page
     expect(page).to have_current_path(candidate_interface_course_choices_site_path(@course_with_multiple_sites.provider.code, @course_with_multiple_sites.code))
   end
+
+  def and_i_should_see_an_account_created_flash_message
+    expect(page).to have_content('Your apply for teacher training account has been created')
```

Comment:
You could hoist this + the duplicate in the controller into a translation file, optional 👍

Reply:
Have done. Thanks!